### PR TITLE
Adds support for Kibana 7.9.0

### PR DESCRIPTION
**Pull request by bowenlan-amzn**
*Issue #, if available:*

*Description of changes:*
Some UTs failed:
- enzyme simulate keyDown with keyCode somehow not working, Tianli found out using `{key: 'Enter'}` can fix this
- in `Monitors.js getActiveAlerts()` method, `alerts` in the response is undefined which failed 2 tests below
  - onClickAcknowledgeModal acknowledges selected alerts for each monitor
  - onBulkEnable calls updateMonitors with selectedItems and enabled:true update
we need to mock the response in the previous UT to fix this

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: public/pages/CreateMonitor/containers/AnomalyDetectors/__tests__/AnomalyDetector.test.js [hunk 1]
```diff
@@ -22,6 +22,11 @@ import AnomalyDetectors from '../AnomalyDetectors';
 import { httpClientMock } from '../../../../../../test/mocks';
 import { AppContext } from '../../../../../utils/AppContext';
 
+import '@testing-library/jest-dom/extend-expect';
+import { render, fireEvent, wait } from '@testing-library/react';
+// @ts-ignore
+import userEvent from '@testing-library/user-event';
```
Comment (dbbaughe) [inline public/pages/CreateMonitor/containers/AnomalyDetectors/__tests__/AnomalyDetector.test.js:28]: Remove these if not used

File: package.json [hunk 2]
```diff
@@ -33,7 +33,10 @@
     "@kbn/expect": "link:../../packages/kbn-expect",
     "@kbn/plugin-helpers": "link:../../packages/kbn-plugin-helpers",
     "husky": "^3.0.0",
-    "lint-staged": "^9.2.0"
+    "lint-staged": "^9.2.0",
+    "@testing-library/jest-dom": "^4.0.0",
+    "@testing-library/react": "^8.0.5",
+    "@testing-library/user-event": "^4.1.0"
```
Comment (dbbaughe) [inline package.json:39]: Remove if not used

**Comment by bowenlan-amzn** [comment]
Test Suites: 6 skipped, 78 passed, 78 of 84 total
Tests:       20 skipped, 366 passed, 386 total
Snapshots:   127 passed, 127 total
Time:        43.683s
Ran all test suites.
✨  Done in 45.39s.

**Comment by dbbaughe** [comment]
I think the release note file is missing for 1.10.0.0

**Pull request merged**